File: Easy/24.SwapCase/solve.rb
def SwapCase(input)

  # code goes here
 	result = []
	input.split("").each do |char|
		/[[:upper:]]/.match(char) ? result << char.downcase : result << char.upcase
	end
	result.join()
         
end
   
# keep this function call here    
puts SwapCase(STDIN.gets)

File: Easy/36.BitwiseOne/solve.py
def BitwiseOne(strArr):
	result = ""
	for i in range(0, len(strArr[0])):
		if strArr[0][i] == "0" and strArr[1][i] == "0":
			result += "0"
		else:
			result += "1"
	return result
    
# keep this function call here  
print BitwiseOne(raw_input())